practice-one/src/components/service.html

```diff
@@ -0,0 +1,7 @@
+<section class="service-container">
+  <div><img src="./assets/images/service.png" alt="Service's illustration"></div>
```

Comment:
this `div` is useless

Reply:
This `div` should act as like frame for image, and the image will scale to that frame, and also that way will prevent layout shift if the image cannot load

Reply:
I added and div container with the same size of image